tracing-futures/src/lib.rs

```diff
@@ -64,6 +64,9 @@ pub trait Instrument: Sized {
     fn instrument(self, span: Span) -> Instrumented<Self> {
         Instrumented { inner: self, span }
     }
+    fn current_instrument(self) -> Instrumented<Self> {
+        Instrumented { inner: self, span: Span::current() }
```

Comment:
Nit, take it or leave it: this _could_ also be implemented as 
```suggestion
        self.instrument(Span::current())
```

This has the advantage of picking up any future changes to the `instrument` method without requiring changes here, although I doubt that'll be an issue. It _does_ potentially introduce an additional stack frame, but we probably should mark both these methods as `#[inline]` regardless...

Up to you if you want to change this or not; I'm happy to merge either way.